zkevm-circuits/src/evm_circuit/execution/sha3.rs

```diff
@@ -48,19 +48,25 @@ impl<F: Field> ExecutionGadget<F> for Sha3Gadget<F> {
 
         let copy_rwc_inc = cb.query_cell();
         let rlc_acc = cb.query_cell();
-        cb.copy_table_lookup(
-            cb.curr.state.call_id.expr(),
-            CopyDataType::Memory.expr(),
-            cb.curr.state.call_id.expr(),
-            CopyDataType::RlcAcc.expr(),
-            memory_address.offset(),
-            memory_address.address(),
-            0.expr(), // dst_addr for CopyDataType::RlcAcc is 0.
-            memory_address.length(),
-            rlc_acc.expr(),
-            cb.curr.state.rw_counter.expr() + cb.rw_counter_offset(),
-            copy_rwc_inc.expr(),
-        );
+        cb.condition(memory_address.has_length(), |cb| {
```

Comment:
pls. update spec accordingly if not yet .

Reply:
Yes, it's included in this fix PR: https://github.com/privacy-scaling-explorations/zkevm-specs/pull/246